packages/workspace/src/node/default-workspace-server.ts

```diff
@@ -81,19 +82,52 @@ export class DefaultWorkspaceServer implements WorkspaceServer {
         this.root.resolve(root);
     }
 
-    getRoot(): Promise<string | undefined> {
+    getWorkspace(): Promise<string | undefined> {
         return this.root.promise;
     }
 
-    async setRoot(uri: string): Promise<void> {
+    async setWorkspace(uri: string): Promise<void> {
```

Comment:
As discussed, I assumed that there were already tests for the workspace server, and that we could add a couple on top. Since that's not the case, I would be ok for this to be a separate task. I think we could do that after the multi-root patch goes is, and then you might also add a test or two to validate the recent WS history.